package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@digicatapult/openapi-ts-template",
+  "version": "0.0.1",
+  "description": "An open api typescript nodejs/express api service template",
+  "main": "src/index.ts",
+  "scripts": {
+    "lint": "eslint .",
+    "depcheck": "depcheck",
```

Comment:
I think there'll need to a be a `.depcheckrc` to ignore some deps that we are using but cause the depcheck to fail https://github.com/digicatapult/wasp-thing-service/blob/main/.depcheckrc